Backend/app/loan_management/manage_loans_service.py

```diff
@@ -250,24 +250,75 @@ def _check_reserved_book(
             )
         return None
 
-    def consult_limit_by_user_email(self, email: str) -> bool:
+    def cant_reserved_loans(self, email: str) -> int:
         cant_reserved_loans = self.query_database(
             """SELECT COUNT(*)
                 FROM loan
                 WHERE userEmail = ?
                 AND loanStatus = ? AND returnDate IS NULL""",
             (email, "reserved"),
         )
+        return cant_reserved_loans[0]
+
+    def cant_loaned_books(self, email: str) -> int:
         cant_loaned_books = self.query_database(
             """SELECT COUNT(*)
                 FROM loan
                 WHERE userEmail = ?
                 AND loanStatus = ? AND returnDate IS NULL""",
             (email, "loaned"),
         )
+        return cant_loaned_books[0]
+
+    def cant_points_user(self, email: str) -> int:
+        points_user = self.query_database(
+            """SELECT points FROM users WHERE email= ?""", (email,)
+        )
+        return points_user[0]
+
+    def limit_amount_of_loans(
+        self,
+        email: str,
+        points_user: int,
+        cant_reserved_loans: int,
+        cant_loaned_books: int,
+    ):
 
-        # TODO: fix
-        if cant_reserved_loans[0] >= 3 or cant_loaned_books[0] >= 3:
+        is_limit_default: bool = cant_reserved_loans == 3 or cant_loaned_books == 3
+
+        if points_user < 100 and is_limit_default:
```

Comment:
This is not how point exchange works. Check how the points store works with licence level updates.